app/assets/stylesheets/application.scss

```diff
@@ -61,3 +61,17 @@ body {
   text-align: center;
 
 }
+
+#map-container {
+   height: 400px;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered border-color, border-radius, border-style, border-width, box-shadow, height, margin-top